assets/sass/06-components/navigation.scss

```diff
@@ -351,10 +351,6 @@
 		.menu-item {
 			margin: calc(2 * var(--primary-nav--padding)) 0;
 
-			&:last-child() {
```

Comment:
Removing this affects both menus, so there is now a space below the sub menus in the responsive version of the menu.

Reply:
Simply add media query for this section like `@include media(mobile) {}`.

Reply:
(mobile is for mobile and and up, mobile-only is for mobile only)

Reply:
Thank you, folks. I am done with the changes.